qa/tasks/scripts/ses_disk_fault_injection.sh

```diff
@@ -0,0 +1,36 @@
+set -ex
+
+# get storage minion
+declare -a disk_storage_minion=("$@")
```

Comment:
why use an array for a single storage node?

Reply:
that's right, but argument will send me list of storage nodes. 
Probably I would need to edit script to choose only one from the list, or to send only one storage node as an argument to the script.

Reply:
in ses_qa.py, you can get a single storage node by doing, e.g. `self.nodes_storage[0]`

Reply:
and the total number of storage nodes is `len(self.nodes_storage)`

Reply:
similar methods can be used to get a single mon node, number of mon nodes, etc.

Reply:
I implemented random in \_\_init\_\_.py in commit edd13d2a1aca8f3092ccc5e9850b68329f8abaf8

Reply:
and random for ses_qa.py and related scripts in commit
2df7ca208c4b020ea5090e11fd9b13d7a329783d